Mathlib/CategoryTheory/Limits/Cones.lean

```diff
@@ -335,6 +335,14 @@ theorem cone_iso_of_hom_iso {K : J ⥤ C} {c d : Cone K} (f : c ⟶ d) [i : IsIs
         w := fun j => (asIso f.hom).inv_comp_eq.2 (f.w j).symm }, by aesop_cat⟩⟩
 #align category_theory.limits.cones.cone_iso_of_hom_iso CategoryTheory.Limits.Cones.cone_iso_of_hom_iso
 
+/-- Given an isomorphism of cones, produce an isomorphism of their points. -/
+@[simps]
+def ptIsoOfIso  {K : J ⥤ C} {c d : Cone K} (i : c ≅ d) : c.pt ≅ d.pt where
```

Comment:
I'm slightly cautious about expanding the API with this one. (i.e. If we introduce it, what lemmas should we be writing, or feel we been later, relating it to other existing things. e.g. if you start with an iso between the points of two limits cones, obtain the iso of cones, and then apply this you are back where you started.) Similarly -- what about the dual?

If you can think of a way to avoid introducing this I'd be perfectly happy with the rest of the PR.

Reply:
One should probably use the "forgetful" functor from cones to the target category instead. We have this, right? I might have time to fix this later today.

Reply:
Ok, `ptIsoOfIso` is gone